web-ui/src/main/resources/catalog/components/viewer/wps/partials/urldiscovery.html

```diff
@@ -2,6 +2,7 @@
   <div class="form-group">
     <input class="form-control input-sm" type="text" data-toggle="dropdown"
            data-ng-model="url" data-ng-change="ctrl.doRequest()"
+           aria-label="{{'serviceUrl' | translate}}"
```

Comment:
do you really want to set the actual URL as aria-label instead of some label like Service URL or something similar?